packages/amplify-cli-core/src/index.ts

```diff
@@ -15,6 +15,7 @@ export * from './isPackaged';
 export * from './cliConstants';
 export * from './deploymentSecretsHelper';
 export * from './deploymentState';
+export * as utils from './utils';
```

Comment:
I would prefer to export `openIfNotCI` directly. Having an exported member like `utils` will likely become a kitchen sink for all kinds of unrelated functions in the future

Also utils is a super overloaded word (as you saw when you had to rename some imports) so I think more specific export names are better

Reply:
I agree util is overused, but exposing this directly would also is ugly. I would have used `helper` but that too is over used. I am open for suggestion if we can namespace this to something meaningful.

Reply:
We could just name it 'open' (or maybe 'cliOpen' if we want to distinguish it from the npm package). We don't need to leak the fact that it's a noop in CI environments in the function name